subcommands/devices/show.go

```diff
@@ -126,6 +126,11 @@ func doShow(cmd *cobra.Command, args []string) {
 	}
 	if device.ActiveConfig != nil {
 		fmt.Println("Active Config:")
+		if len(device.ActiveConfig.CreatedBy) > 0 {
+			user, err := api.UserAccessDetails(factory, device.ActiveConfig.CreatedBy)
+			subcommands.DieNotNil(err)
```

Comment:
It is OK to have a config which was created by a user which was removed.
So, I'd show smth like `<deleted user>` or `<unknown user>` and a user ID in this case, rather than erroring out.

That said, I don't know all the details.
Maybe, we keep users info forever, even when they are technically deleted.